templates/books/articles/audiobooksby-database.html

```diff
@@ -75,6 +79,7 @@ <h2>Схема data.json</h2>
   // list of Person.uuid
   narrators: string[];
   links: Links[];
+  duration: string;
```

Comment:
Is it string or number? The old duration_sec was number.

Reply:
Unless I missed custom serialization logic somewhere...  Django's `DurationField` must be serialized to string:

venv/lib/python3.9/site-packages/django/utils/duration.py
```
def duration_string(duration):
    """Version of str(timedelta) which is not English specific."""
    days, hours, minutes, seconds, microseconds = _get_duration_components(duration)

    string = '{:02d}:{:02d}:{:02d}'.format(hours, minutes, seconds)
    if days:
        string = '{} '.format(days) + string
    if microseconds:
        string += '.{:06d}'.format(microseconds)

    return string
```

But I may be wrong.

Reply:
Yeah, make sense to keep it as number of seconds. Let me fix that.